packages/story-editor/src/components/publishModal/content/storyPreview.js

```diff
@@ -189,27 +181,74 @@ const Publisher = styled(Text).attrs({
   color: ${({ theme }) => theme.colors.fg.secondary};
 `;
 
-// TODO https://github.com/GoogleForCreators/web-stories-wp/issues/10584
-const ENABLE_EDIT_FEATURED_MEDIA = false;
-
 const StoryPreview = () => {
-  const { title, featuredMedia, publisherLogo } = useStory(
-    ({ state: { story } }) => ({
+  const { title, featuredMedia, publisherLogo, updateStory } = useStory(
+    ({ state: { story }, actions }) => ({
       title: story?.title,
       featuredMedia: story?.featuredMedia || {},
       publisherLogo: story?.publisherLogo || {},
+      updateStory: actions.updateStory,
     })
   );
 
-  const publisher = useConfig(({ metadata }) => metadata?.publisher);
-  const hasUploadMediaAction = useConfig(
-    ({ capabilities }) => capabilities?.hasUploadMediaAction
+  const {
+    allowedImageFileTypes,
+    allowedImageMimeTypes,
+    hasUploadMediaAction,
+    publisher,
+  } = useConfig(
+    ({
+      allowedImageFileTypes,
+      allowedImageMimeTypes,
+      capabilities,
+      metadata,
+    }) => ({
+      allowedImageFileTypes,
+      allowedImageMimeTypes,
+      hasUploadMediaAction: capabilities?.hasUploadMediaAction,
+      publisher: metadata?.publisher,
+    })
   );
 
   // Honor 2:3 aspect ratio that cover previews have
   const mediaWidth = 232;
   const mediaHeight = Math.round(mediaWidth / PAGE_RATIO);
 
+  const handleChangePoster = useCallback(
```

Comment:
All taken from https://github.com/GoogleForCreators/web-stories-wp/blob/928311c162f7d7f0454364755e689ef099badb33/packages/wp-story-editor/src/components/documentPane/publish/publish.js#L171-L191

Reply:
thanks for the reference :)